apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1/ComputeEnumConstants.g.cs

```diff
@@ -0,0 +1,5204 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Google.Cloud.Compute.V1
+{
+    /// <summary>Helper constants with the wire representation for enums within the Compute API.</summary>
+    public static class ComputeEnumConstants
+    {
+
+        /// <summary>Container class for enums within the <see cref="global::Google.Cloud.Compute.V1.AccessConfig"/> message.</summary>
+        public static class AccessConfig
+        {
+
+            /// <summary>Constants for wire representations of the <see cref="global::Google.Cloud.Compute.V1.AccessConfig.Types.NetworkTier"/> enum.</summary>
+            public static class NetworkTier
+            {
+
```

Comment:
I left two comments on the generator that I think will remove this blank line...